am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -28,6 +29,23 @@ public static ExplicitAccessRecord createRecord(String resourceId,
             .build();
     }
 
+    public static ExplicitAccessRecord createAttributeRecord(String resourceId,
```

Comment:
There is lots of duplicated code. Extend existing `createRecord` method. Overload if needed (having new method name for same in this case is bad idea IMO)

Reply:
👍